src/Tribe/Commerce/Warnings.php

```diff
@@ -0,0 +1,59 @@
+<?php
+/**
+ * Conditional Warnings on Ticket Features
+ *
+ * Class Tribe__Tickets__Commerce__Warnings
+ *
+ * @since TBD
+ *
+ */
+class Tribe__Tickets__Commerce__Warnings {
+
```

Comment:
:no_entry_sign: Class name is not valid; consider Tribe_Tickets_Commerce_Warnings instead

<sub>hash: 72b355b328bee46df7ccb1683ddcd4b3</sub>